src/solver/conv/conv_direct_naive_conv.cpp

```diff
@@ -361,11 +361,27 @@ GetConv2DFWDSolution(const ExecutionContext& ctx, const ::miopen::conv::ProblemD
     size_t grid_size  = 1;
     if(problem.IsLayoutDefault())
     {
-        grid_size = static_cast<size_t>(n) * k;
+        size_t all_workload = static_cast<size_t>(n) * k;
+        if(all_workload <= block_size)
+        {
+            grid_size = all_workload;
+        }
+        else
+        {
+            grid_size = (all_workload + block_size - 1) / block_size;
```

Comment:
See https://github.com/ROCm/MIOpen/issues/2748
It is an integer `Ceil()` function.
It's just a reminder that the problem still exists.